### PR TITLE
Adding initialization for remote_debugging_enabled_ (from issue 274840)

### PR DESCRIPTION
**Pull request by terriko**

### CODE REVIEW

File: runtime/browser/runtime.cc [hunk 1]
```diff
@@ -60,6 +60,7 @@ Runtime::Runtime(content::WebContents* web_contents)
     : WebContentsObserver(web_contents),
       web_contents_(web_contents),
       fullscreen_options_(NO_FULLSCREEN),
+      remote_debugging_enabled_(false),
```
Comment (pozdnyakov) [inline runtime/browser/runtime.cc]: we never use this data member, it should be just removed from the class declaration

Comment (terriko) [inline runtime/browser/runtime.cc]: That's even better, if it's possible.

**Comment by terriko** [comment]
Modified code to remove the variable instead as per @pozdnyakov 's comment

**Comment by rakuco** [comment]
Please make sure this is cherry-picked (with `git cherry-pick -x`) to crosswalk-12 and crosswalk-11.

**Comment by baleboy** [comment]
@rakuco crosswalk-12 definitely, but in crosswalk-11 for some reason the variable was initialized (though I'm not sure if it's used)

**Comment by rakuco** [comment]
Oh, right. I forgot the shared process model changes did not go into crosswalk-11.

**Comment by rakuco** [comment]
Done in #2864.

**Pull request merged**